Valentina-K/project-pet-shelter-new
# created structure, add environment variables, fix vite.config

### **Description of Changes**
I have added environment variables, configured Redux for ads, photos, authentication, and categories, integrated Browser Router, and created the homepage.

### **What problem is being solved?**
The PR integrates core functionalities for managing ads, photos, user authentication, and category handling. It also sets up routing for better navigation and adds the homepage.

### **What are the main changes?**
Added environment variables
Configured Redux for ads, photos, authentication, and categories
Integrated Browser Router for navigation
Created homepage layout and routing

### **Type of Changes**
 New feature ✨
 
### **Related Issues**
Closes #<issue-number>

### **How to Test**
Pull the branch and install dependencies
Add environment variables
Start the app and check the homepage
Test Redux flows for ads, photos, and authentication
Verify that routing works correctly

### **Screenshots / Demo**
N/A

### **Checklist**

- Code adheres to the project's guidelines

- Testing has been completed successfully

- Documentation has been added/updated

- All new and existing tests pass

### **Additional Information**
None
